### PR TITLE
Fix: Do not configure deprecated fixers

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] stops configuring deprecated and configures replacement fixers for `friendsofphp/php-cs-fixer`

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -41,6 +41,10 @@
             ],
         ],
         'blank_line_after_namespace' => true,
+        'blank_lines_before_namespace' => [
```
Comment (localheinz) [inline .php-cs-fixer.dist.php:44]: See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.23.0/doc/rules/namespace_notation/blank_lines_before_namespace.rst.

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -300,6 +300,9 @@
         'single_space_around_construct' => true,
         'single_trait_insert_per_statement' => true,
         'space_after_semicolon' => true,
+        'spaces_inside_parentheses' => [
```
Comment (localheinz) [inline .php-cs-fixer.dist.php:304]: See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.23.0/doc/rules/whitespace/spaces_inside_parentheses.rst.

File: .php-cs-fixer.dist.php [hunk 3]
```diff
@@ -319,6 +322,11 @@
             ]
         ],
         'trim_array_spaces' => true,
+        'type_declaration_spaces' => [
```
Comment (localheinz) [inline .php-cs-fixer.dist.php:326]: See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.23.0/doc/rules/whitespace/type_declaration_spaces.rst.

**Comment by sebastianbergmann** [comment]
Can you please review the merges into `9.6`, `10.3`, and `main`? I ran into merge conflicts along the way and want to make sure I resolved them correctly. Thanks!

**Comment by localheinz** [comment]
Will do tomorrow morning, @sebastianbergmann!

**Comment by localheinz** [comment]
Looks good to me, @sebastianbergmann - nothing to do here!

**Pull request merged**